app/javascript/components/patient/assessment/steps/SymptomsAssessment.js

```diff
@@ -265,12 +267,12 @@ class SymptomsAssessment extends React.Component {
         <Card.Body>
           {this.props.current_user && (
             // Only allow users, not monitorees, to edit the report date
-            <Form.Group>
+            <Form.Group controlId="reported_at">
               <Form.Row>
                 <Form.Label className="input-label mr-2 pt-1">Symptom Report for Date:</Form.Label>
                 <DateInput
                   id="reported_at"
-                  date={this.props.assessment.reported_at ?? moment().toDate()}
+                  date={moment(this.state.reportState.reported_at, 'YYYY-MM-DD HH:mm Z').format('YYYY-MM-DD HH:mm Z')}
```

Comment:
I'm getting an "Invalid time value" error again in Firefox with this change from your most recent commit.

Reply:
https://github.com/SaraAlert/SaraAlert/pull/1248/commits/f9feb996a84a24f6ef25277173bc8d3a6aadca73